utilities/logic/async_wrapper.py

```diff
@@ -70,6 +70,49 @@ def daemonize_self():
     os.dup2(dev_null.fileno(), sys.stdout.fileno())
     os.dup2(dev_null.fileno(), sys.stderr.fileno())
 
+# FUTURE: cloned from ActionBase. Move this to module_utils to share between core/async_wrapper, AnsibleModule-ify this module so it's Ansizballz-compatible
```

Comment:
If you follow my suggestion in https://github.com/ansible/ansible/pull/17834 to move the function out of action/__init__.py and into module_utils now, update this comment to point to the file that the code lives in.  Basically, we want to let future maintainers know that they have to make any changes in both places.